src/main/protobuf/build/buildfarm/v1test/buildfarm.proto

```diff
@@ -90,21 +90,14 @@ message InstanceConfig {
   }
 }
 
-message CASInsertionControl {
-  // the size limit of inline content
-  int64 limit = 1;
+enum CASInsertionPolicy {
+  UNKNOWN = 0;
 
-  enum Policy {
-    UNKNOWN = 0;
+  ALWAYS_INSERT = 1;
 
-    ALWAYS_INSERT = 1;
+  INSERT_ABOVE_LIMIT = 2;
 
-    INSERT_ABOVE_LIMIT = 2;
-
-    NEVER_INSERT = 3;
-  }
-
-  Policy policy = 2;
+  NEVER_INSERT = 3;
 };
 
 message WorkerConfig {
```

Comment:
You don't persist these protos anywhere, right?

Reply:
Not in binary form - these are all intended to be loaded like CROSSTOOL